docs/using/components/configuration/configuration.rst

```diff
@@ -50,21 +50,44 @@ file type where the information is stored or a number of other sections.
 
 .. jsonschema:: schemas/model.yml
 
-All types of ``structure`` definitions have two keywords in common: ``v_inner_boundary`` and ``v_outer_boundary``.
+Abundances
+^^^^^^^^^^
+The ``abundance`` section has a possible ``file`` parameter with ``type`` (currently only ``artis`` is allowed)
+and a ``name`` parameter giving a path to a file containing the abundance information.
+
+.. jsonschema:: schemas/model_definitions.yml#/definitions/abundances/file
+
+.. jsonschema:: schemas/model_definitions.yml#/definitions/abundances/uniform
+
 In the ``structure`` section, one can specify a ``file`` section containing a ``type`` parameter
 (currently only ``artis`` is supported``) and a ``name`` parameter giving a path top a file.
 
 If one doesn't specify a ``file`` section, the code requires two sections (``velocities`` and ``densities``) and a
 parameter ``no_of_shells``. ``no_of_shells`` is the requested number of shells for a model. The ``velocity`` section
 requires a ``type``. Currently, only ``linear`` is supported.
 
```

Comment:
Ctrl+X "In the ``structure`` section .... only ``linear`` is supported." and Ctrl+V right after "All types of ``structure`` definitions have two keywords in common: ``v_inner_boundary`` and ``v_outer_boundary``."

Reply:
Yes, sorry about bothering you. I was a little bit confused about the above changes that were requested prior to these. I will change them accordingly 😉

Reply:
Done :)